### ISSUE TITLE
[Link and button labels] If your content has multiple CTAs, the CTAs don't have a unique descriptive label and/or give the user the intended result.  (08.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59465]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
If your content has multiple CTAs, the CTAs don't have a unique descriptive label and/or give the user the intended result. 

### Issue Details
When a user expands the "additional filters" accordion, a second instance of the "filter" button appears. This second button does not have unique button text and appears to be visually and functionally duplicative. This introduces unnecessary cognitive load to the user in trying to discern the difference and decide which one should be clicked.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/51115-Sara-080101.png

### VA.gov Experience Standard
[Category Number 08, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Eliminate the duplicate button. Allow a single filter button to take action on selections made in the search bar or any dropdown filter. This will also create better alignment with design system guidance on primary button usage, which should be reserved for the most important action you want the user to take on the page.

Note that the single filter button should be placed *after* the additional filters. Two reasons for this:

- If the filter button appears before the additional filters, screen reader users may never realize those additional filters exist (reach the filter button, assume that's the end of the filter form, hit enter).
- If the filter button appears before the additional filters, keyboard-only users will have to tab past the button, set their filter criteria, then [shift] + [tab] back to the filter button. In general we don't want to force users to tab back and forth to accomplish a single task.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
